extensions/amp-list/0.1/test/test-amp-list-load-more.js

```diff
@@ -42,6 +42,7 @@ describes.realWin(
     let ampdoc;
     let element, list;
     let templates;
+    let lockHeightSpy, unlockHeightSpy;
```

Comment:
nit: would be nice to have a test ensuring that `load-more` is not enabled with `layout=container`